bundles/org.openhab.binding.oppo/src/main/java/org/openhab/binding/oppo/internal/OppoBindingConstants.java

```diff
@@ -38,6 +38,8 @@
     public static final Integer BDP10X_PORT = 48360;
     public static final Integer BDP20X_PORT = 23;
 
+    public static final String FIRMWARE_PROPERTY = "Firmware Version";
```

Comment:
Please use the existing constant for this: https://github.com/openhab/openhab-core/blob/5ab248c158164bf68f020192e98d0d4ccb8010fc/bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/Thing.java#L55

Reply:
done